docs/src/app/(docs)/uploading-files/page.mdx

```diff
@@ -99,16 +99,61 @@ Once your backend adapter has received and validated the request, you will next
 need to generate presigned URLs. First, generate some file keys for the files to
 be uploaded.
 
-To generate a file key, generate a sha256 hex digest of a file seed of your
-choice, as well as of your API key. Concatenate the API key and the file seed
-with a `:` separator and encode the result as a base64 string.
+To generate a file key, generate a [Sqids](https://sqids.org/) id of a file seed
+of your choice, as well as of your appId. Concatenate the two Sqids to form the
+final key.
 
-```ts
-const fileSeed = `${file.name}${file.size}`; // can be anything
-const seedHash = sha256Hex(fileSeed);
-const apiKeyHash = sha256Hex(apiKey);
+The file seed can be anything you want, but it should be unique for each file.
 
-const fileKey = encodeBase64(`${apiKeyHash}:${seedHash}`);
+```ts
+import SQIds, { defaultOptions } from "sqids";
+
+// A simple function to shuffle the alphabet for the Sqids
+function shuffle(str: string, seed: string) {
+  const chars = str.split("");
+  const seedNum = Hash.string(seed);
+
+  let temp: string;
+  let j: number;
+  for (let i = 0; i < chars.length; i++) {
+    j = ((seedNum % (i + 1)) + i) % chars.length;
+    temp = chars[i];
+    chars[i] = chars[j];
+    chars[j] = temp;
+  }
+
+  return chars.join("");
+}
+
+function generateKey(
+  file: FileProperties,
+  appId: string,
+) =>
+  // Get the parts of which we should hash to constuct the file seed
+  // This allows the user to customize the hashing algorithm
+  // If they for example want to generate the same key for the
+  // same file whenever it was uploaded
+  const hashParts = JSON.stringify(
+    [
+      file.name,
+      file.size,
+      file.type,
+      file.lastModified,
+      Date.now(),
+    ],
+  );
```

Comment:
This feels a bit redundant for the docs no? Just pass in the final string as arg maybe to make the function more concise and only show the "important" parts that are verified?